CONTRIBUTING.rst

```diff
@@ -11,6 +11,13 @@ Setting up the environment
 4. Install `pre-commit <https://pre-commit.com/>`_
 5. Run ``pre-commit install`` to install pre-commit hooks
 
+.. tip::
```

Comment:
Does this render properly on GitHub?

If not, just use a subsection or bold text. It's not as pretty on our page but it should work when viewed on GitHub as well since it's linked there

Reply:
For some definition of "properly" :stuck_out_tongue: You can see how this looks at https://github.com/starlite-api/starlite/blob/77c56f795b1bdcbc5d94490c22f4331923f59cc4/CONTRIBUTING.rst.

It's unstyled, but much better than say, admonitions for MkDocs would be which GitHub doesn't support at all.

Happy to change this to regular rst without the directive if you feel this still looks too badly on plain GitHub previews. Let me know what you'd prefer! :smile:

Reply:
Okay, it's good enough so we can keep it.